### PR TITLE
feat(keycloak): setup docker to tests

### PR DESCRIPTION
**Pull request by renatoaug**
gif-or-image

### What?

Setup docker and create a service for `validate access token` to test the docker.

### Why?

Because we need this to test next features.

### CODE REVIEW

File: packages/keycloack/src/service/validate-acess-token.service.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common'
+import { KeycloakClient } from '../client'
+
+@Injectable()
+export class ValidateAccessToken {
```
Comment (ndeitch) [inline packages/keycloack/src/service/validate-acess-token.service.ts:5]: What do you think of adding service to class name?
```suggestion
export class ValidateAccessTokenService {
```

**Review by ndeitch — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM node:10.20.1-alpine3.11 As dev
+
+WORKDIR /usr/src/app
+
+RUN apk add --no-cache bash curl
+
+COPY package*.json ./
+
+RUN ls -la
```
Comment (djaircarvalho) [inline Dockerfile:9]: I think it can be removed

**Review by djaircarvalho — APPROVED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
+FROM node:10.20.1-alpine3.11 As dev
+
+WORKDIR /usr/src/app
+
+RUN apk add --no-cache bash curl
+
+COPY package*.json ./
```
Comment (djaircarvalho) [inline Dockerfile:7]:
```suggestion
COPY package*.json ./
```

File: docker-compose.yml [hunk 4]
```diff
@@ -19,3 +19,19 @@ services:
...
+      - KEYCLOAK_IMPORT=/tmp/realm.json
+    networks:
+      - nestjs-keycloak-network
+    ports:
+      - 8080:8080
+    volumes:
+      - ${PWD}/realm.json:/tmp/realm.json
+
+networks:
+  nestjs-keycloak-network:
```
Comment (djaircarvalho) [inline docker-compose.yml:37]: I think we can remove network since none of the services communicate with each other.

**Review by djaircarvalho — APPROVED** [review]

**Pull request merged**